src/web/docusaurus/docs/getting-started/environment-setup.md

```diff
@@ -262,6 +268,12 @@ This is the default setting, you do not need to copy or modify any `env` file.
 pnpm services:start
```

Comment:
Like @Genne23v said, hopefully it's the last change :))
```suggestion
pnpm services:start parser 
```

Reply:
I don't think this is a good idea. This section (Option 1) is about running telescope locally, the whole app. To run mocriservices individually, we have section 4, which already mentions how to run just one service, chech [here](https://telescope.cdot.systems/docs/getting-started/environment-setup#option-4-run-microservices-individually).

I think what @Genne23v  meant is the mistake in the last line of this section:
> update `pnpm:services start parser` to `pnpm services:start parser`